.github/workflows/lint.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           linter: [copyright, flake8, pep257]
     steps:
     - uses: actions/checkout@v2
-    - uses: ros-tooling/action-ros-lint@0.1.0
+    - uses: ros-tooling/action-ros-lint@v0.1
```

Comment:
you probably figured it out, but you're using an Eloquent docker image, but by default `action-ros-lint` uses rolling for `distribution`, which explains not being able to locate `ros-rolling-ament-...` :grin: